### PR TITLE
Updates to InstrumentModelEnum

### PR DESCRIPTION
**Pull request by aclum**
Adding aliases and structured aliases.

# Guidelines

## _Soft_ Schema Freeze

The `nmdc-schema` and `berkeley-schema-fy24` schemas are under a soft freeze, which means changes **should not** be made that have any downstream implications. To ensure this, all PRs created during the freeze will be closely reviewed with **every** component of the NMDC system in mind.

## Reviewers

To ensure no changes are made unexpectedly, PR creators will use this PR template to tag and notify all task coordinators. Review should be specifically requested from _all_ [Berkeley Schema Roll Out task coordinators](https://docs.google.com/document/d/1XXN1YuaBuSkxPXeiLKm5YxYzXTamBPQrzzeLhlh7PWs/edit#heading=h.u52g8v319adh) that you expect to be affected by this PR.

We expect task coordinators to review PRs and provide feedback/approval within 1 week of when they are identified as reviewers. 

PRs will **NOT** be merged until all task coordinators (or their delegates) have approved it; either here on GitHub (via "`Review changes` > `Approve`" or an equivalent comment) or verbally.

Expedition, questions, and discussion can happen at any meeting.

Delays in review & merging should be addressed in meetings or with NMDC leadership.

| If you expect the changes to<br>impact this component... | ...[request a review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review)<br>from this person |
| -- | -- |
| Metadata<br>Schema | @mslarae13 |
| Runtime<br>Mongo database<br>Database migrations | @eecavanna,<br>who will pull in<br>@shreddd as needed |
| Postgres<br>Ingest | @naglepuff |
| Data Portal | @aclum |
| Workflows: MG & MT | @mbthornton-lbl |
| Workflows: MetaB & NOM | @corilo |
| Workflows: LipidO | @kheal |
| Workflows: MetaP | @SamuelPurvine |
| ETL code | @sujaypatil96 |
| Jupyter notebooks | @brynnz22 |

# PR Information

## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation
- [x] Schema change: Structure and content
  - created, updated, or deleted a `class`, `slot`, or `enum`
  - changed whether a `slot` is `multivalued`
  - changed the way a `slot` is assigned to a `class`
  - changed the `permissible_values` of an `enum`
  - _etc._
- [x] Schema change: Cleanup and preparation
  - updated the description of a `class`, `slot`, or `enum`
  - updated the `mappings` of a `class`, `slot`, or `enum` to an ontology
  - added an `enum` for _future_ use (it is not in the `range` of any `slot`)
  - _etc._
     
## Description

This PR expands InstrumentModel permissible values, adds structured aliases, and adds a mapping file required by the GOLD ETL code (MVP for rollout).  This unblocks https://github.com/microbiomedata/nmdc-runtime/issues/622

## Related Issues

> All PRs should relate to or fix an issue(s). Please identify the issue(s) below.

- Related Issue(s): https://github.com/microbiomedata/nmdc-runtime/issues/622
- Fixes: https://github.com/microbiomedata/nmdc-schema/issues/2174

## Did you add/update any tests?

- [ ] Yes
- [x] No _(Add a justification below)_
- [ ] I need help with writing tests

mostly adding aliases

## Could this schema change make it so any valid data becomes invalid?

> This is a question about what the schema allows. It is not a question about what happens to exists in the NMDC database right now.
> 
> Example: If, in this PR branch, you renamed a `slot` from `foo` to `foo_bar`, the answer to this question would be "yes," **even if** nothing in the NMDC database _currently_ uses the `foo` `slot`.
>
> More examples: `slot` or `class` name changes, changes to a `slot`'s `multivalued` state, changes to a `slot`'s `range` (e.g. string to integer), changes to `slot` assignments to `class`es, changes to an `enum`'s `permissible_values`

- [ ] Yes _(A migrator is required)_
- [ ] No
- [x] I need help determining this
@brynnz22  not sure if we want to use these new structured aliases for the migrator instead of updating in two places.

## If you answered "Yes", does this PR branch include that migrator?

- [ ] Yes
- [ ] No, this PR is incomplete and I need help writing the migrator

## Does this PR have any downstream implications?

> Examples: any change here that requires a change to workflows, workflow automation, the Mongo-to-Postgres ingest process, Jupyter notebooks, the Runtime, etc.

- [ ] Yes _(Explain below)_
- [x] No

### CODE REVIEW

**Comment by aclum** [comment]
@turbomam  it doesn't like some of the syntax for structured aliases but i followed your example which matches the documentation. I checked and we are using linkml version 1.8.1 which should support these slots.

**Review by sujaypatil96 — APPROVED** [review]
Great PR @aclum 🚀 I checked the StructuredAlias mappings you've specified on each of the permissible values on `InstrumentModelEnum` and everything looks good.

But super strange error with `gen-linkml` complaining about the syntax (`ValueError`) we've followed to implement `structured_aliases` on these permissible values, I can't seem to tell why it's erroring out either. I'll look in more detail.

Edit: <s>great observation about `alias_contexts` having a range of `Uri`, that could be it.</s>

Oh, it's erroring out with an `Unknown argument` error, meaning it's not picking up on that being part of `StructuredAlias`, hm.

**Review by sierra-moxon — APPROVED** [review]
thank you for the linkml ticket.

**Comment by eecavanna** [comment]
I have no concerns about what's in the diff; with the caveat that I'm not too familiar with the subject matter.

**Review by turbomam — APPROVED** [review]

File: assets/misc/gold_seqMethod_to_nmdc_instrument_set.tsv [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#GOLD SeqMethod	NMDC instrument_set
```
Comment (sujaypatil96) [inline assets/misc/gold_seqMethod_to_nmdc_instrument_set.tsv:1]: Why is there a `#` in the header line?

Comment (aclum) [reply assets/misc/gold_seqMethod_to_nmdc_instrument_set.tsv:1]: makes it easier to skip header columns. I can remove if you don't want it.

Comment (sujaypatil96) [reply assets/misc/gold_seqMethod_to_nmdc_instrument_set.tsv:1]: Can we remove the `#`? We should be only be using `\t` as the delimiter.

**Review by brynnz22 — APPROVED** [review]

**Pull request merged**